corehq/apps/userreports/tests/test_report_config.py

```diff
@@ -122,204 +121,6 @@ def test_missing_column_id(self):
             config.validate()
 
 
-class ReportConfigurationTotalRowTest(SimpleTestCase):
-
-    def test_totaling_with_aggregation(self):
-        config_agg = ReportConfiguration(
-            aggregation_columns=['agg_col'],
-            columns=[
-                {
-                    "field": "agg_col",
-                    "aggregation": "simple",
-                    "type": "field",
-                    "calculate_total": False,
-                },
-                {
-                    "field": 'col_1',
-                    "aggregation": "simple",
-                    "type": "field",
-                    "calculate_total": False,
-                },
-                {
-                    "field": 'col_2',
-                    "aggregation": "simple",
-                    "type": "field",
-                    "calculate_total": True,
-                },
-            ],
-        )
-        self.assertEqual(
-            ['Total', '', 8],
-            get_total_row(
-                [
-                    {
-                        'agg_col': 'agg1',
-                        'col_1': 2,
-                        'col_2': 3,
-                    },
-                    {
-                        'agg_col': 'agg2',
-                        'col_1': 4,
-                        'col_2': 5,
-                    },
-                ],
-                config_agg.aggregation_columns,
-                config_agg.report_columns,
-                {}
-            )
-        )
-
-    def test_totaling_without_aggregation(self):
```

Comment:
added some more coverage. originally wasn't planning to, but adding more tests turned out a lot easier than writing the first one